### DISCUSSION TITLE
What's the difference between aqua and other version managers?

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [aquaproj/aqua#474] [Deprecated (Question and Answer)] [answered]
> Note that this is an example of question.

What's the difference between aqua and other version managers?

**Comment by suzuki-shunsuke** [comment]
Please see the document.

* https://aquaproj.github.io/docs/comparison/common
* https://aquaproj.github.io/docs/comparison/asdf
* etc

**Reply by suzuki-shunsuke** [reply]
This answer is old. Please see https://github.com/orgs/aquaproj/discussions/474#discussioncomment-5866913

**Comment by dudicoco** [comment]
@suzuki-shunsuke can you please add [tea](https://github.com/teaxyz/cli) to the comparison?

**Reply by suzuki-shunsuke** [reply]
Thank you for your suggestion.
Unfortunately, I'm not familiar with tea at all so I can't compare tea with aqua at the moment.
But I think at least aqua has the following pros.

- [Easy to use](https://aquaproj.github.io/docs/#easy-to-use)
  - Easy to add new packages to Standard Registry
  - Easy to create your own Registries (local Registry and github_content Registry)
- [Continuous update by Renovate](https://aquaproj.github.io/docs/#continuous-update-by-renovate)
- [Support private packages](https://aquaproj.github.io/docs/guides/private-package)

**Comment by suzuki-shunsuke** [comment, marked as answer]
[The previous answer](https://github.com/orgs/aquaproj/discussions/474#discussioncomment-1809878) was old.
We removed the comparison pages and summarized them to https://aquaproj.github.io/docs/#comparison because it was hard to maintain previous comparison pages and they didn't explain the benefit of aqua shortly so users couldn't understand the benefit of aqua easily.

We summarized the benefit of aqua shortly in https://aquaproj.github.io/docs/#comparison .
